### PR TITLE
Hotfix build mac os

### PR DESCRIPTION
**Pull request by cvetaevvitaliy**
fix build OpenTX for MacOS
I got the same error, the problem is in the python script
https://github.com/opentx/opentx/issues/7549

### CODE REVIEW

**Comment by raphaelcoeffic** [comment]
@cvetaevvitaliy your "fix" is not really a hotfix, rather a rewrite of functions into a worse way... You need to identify the exact thing that makes the change work, then we can talk about it.

In my case, it compiles when I remove the GCC includes (`${arm_gcc_include_path}`):
```diff
diff --git a/radio/src/targets/horus/CMakeLists.txt b/radio/src/targets/horus/CMakeLists.txt
index 525c8a4ad..1e036e929 100644
--- a/radio/src/targets/horus/CMakeLists.txt
+++ b/radio/src/targets/horus/CMakeLists.txt
@@ -329,7 +329,7 @@ if(PYTHONINTERP_FOUND)
   add_custom_command(
     OUTPUT ${PROJECT_BINARY_DIR}/radio/src/datacopy.cpp
     WORKING_DIRECTORY ${RADIO_DIRECTORY}/src
-    COMMAND ${PYTHON_EXECUTABLE} ${RADIO_DIRECTORY}/util/generate_datacopy.py datastructs.h -DPCBHORUS -DPCBX10 -DCPUARM -DSTM32F4 -DSTM32F429_439xx -DCOLORLCD -DBACKUP -DSIMU -I. -Itargets/horus -Itargets/common/arm/stm32 -I${THIRDPARTY_DIR} -I${THIRDPARTY_DIR}/STM32F4xx_DSP_StdPeriph_Lib_V1.4.0/Libraries/CMSIS/Device/ST/STM32F4xx/Include ${arm_gcc_include_path} > ${PROJECT_BINARY_DIR}/radio/src/datacopy.cpp
+    COMMAND ${PYTHON_EXECUTABLE} ${RADIO_DIRECTORY}/util/generate_datacopy.py datastructs.h -DPCBHORUS -DPCBX10 -DCPUARM -DSTM32F4 -DSTM32F429_439xx -DCOLORLCD -DBACKUP -DSIMU -I. -Itargets/horus -Itargets/common/arm/stm32 -I${THIRDPARTY_DIR} -I${THIRDPARTY_DIR}/STM32F4xx_DSP_StdPeriph_Lib_V1.4.0/Libraries/CMSIS/Device/ST/STM32F4xx/Include > ${PROJECT_BINARY_DIR}/radio/src/datacopy.cpp
     DEPENDS ${RADIO_DIRECTORY}/src/datastructs.h ${RADIO_DIRECTORY}/util/generate_datacopy.py
     )
   add_custom_target(datacopy
```

**Comment by cvetaevvitaliy** [comment]
Yes that's right, I rolled back to the previous version generate_datacopy.py

I find problem in CMake 
execute_process(COMMAND "${THE_PROGRAM}" argument OUTPUT_QUIET)

File: radio/src/targets/horus/CMakeLists.txt [hunk 1]
```diff
@@ -294,7 +294,7 @@ MACRO(GET_GCC_INCLUDE_PATH gcc_include_path)
   if (WIN32)
     execute_process(COMMAND arm-none-eabi-gcc -v -x c -E NUL ERROR_VARIABLE _gcc_output OUTPUT_QUIET)
   else()
-    execute_process(COMMAND arm-none-eabi-gcc -v -x c -E - INPUT_FILE /dev/null ERROR_VARIABLE _gcc_output OUTPUT_QUIET)
+    execute_process(COMMAND arm-none-eabi-gcc argument -v -x c -E - INPUT_FILE /dev/null ERROR_VARIABLE _gcc_output OUTPUT_QUIET)
```
Comment (bsongis) [inline radio/src/targets/horus/CMakeLists.txt:297]: I don't see what is this "argument"?

Comment (bsongis) [reply radio/src/targets/horus/CMakeLists.txt:297]: if it was something for cmake, it should be ARGUMENT, but I don't see this in the documentation of execute_process:
https://cmake.org/cmake/help/v3.18/command/execute_process.html

**Pull request closed without merging**